app/lib/core/models/receiver_info.dart

```diff
@@ -22,6 +22,25 @@ class ReceiverInfo {
       @required this.destinationAddress,
       @required this.receiverContactInfo});
 
+  ReceiverInfo.fromJson(Map<String, dynamic> json)
+      : receiverCompanyName = json['receiverCompanyName'],
+        receiverName = json['receiverName'],
+        accountId = json['accountId'] ?? Optional.empty(),
```

Comment:
- [ ] Additionally, this should be:
`accountID = Optional.of(json[`accountID`]`